apps/mobile/src/libs/components/background/components/background-image/background-image.tsx

```diff
@@ -0,0 +1,21 @@
+import {
+	Image,
+	ImageSourcePropType,
+	type ImageStyle,
+	type StyleProp,
+} from "react-native";
+
+import { styles } from "./styles";
+
+const circles =
+	require("../../../../../assets/images/circles.png") as ImageSourcePropType;
```

Comment:
```suggestion
	require("~/assets/images/circles.png") as ImageSourcePropType;
```

Reply:
If I use 

`<Image source={require("~/assets/images/top-circle.png")} style={[styles.image, styles.topImage]} />`

received Error

ESLint: Unsafe assignment of an `any` value. (@typescript-eslint/no-unsafe-assignment)

I have no idea how  to fix it except 

`const circles =
	require("~/assets/images/circles.png") as ImageSourcePropType;`

Do you have any hints?